.drone.yml

```diff
@@ -11,6 +11,14 @@ services:
     image: percona/percona-server:5.6
     environment:
       - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+  selenium:
+    image: selenium/standalone-chrome:3
```

Comment:
I think better to use `image: selenium/standalone-chrome-debug:3.141.59-oxygen` image